content/2_Bronze/Intro_Sorting.mdx

```diff
@@ -170,6 +170,9 @@ Two ways to avoid this:
 
 </JavaSection>
 
+<PySection>
+Python does not have Static Arrays. See Dynamic Arrays for information on sorting a list.
```

Comment:
```suggestion
Python does not have static arrays. See "Dynamic Arrays" for information on sorting a list.
```

unless there is a reason to have it capitalized? and is dynamic arrays a module?

Reply:
pretty sure he's referring to the section underneath
![image](https://user-images.githubusercontent.com/8647920/130401906-44a802fb-16b4-4f9f-8de9-8b72a6de874b.png)

Reply:
that being said though, i think "static arrays" and "dynamic arrays" should be lowercase